connect_ext_ppr/models/configuration.py

```diff
@@ -20,9 +20,9 @@ class Configuration(Model):
         default=ConfigurationStateChoices.INACTIVE,
     )
     created_at = db.Column(db.DateTime(), default=datetime.utcnow)
-    created_by = db.Column(db.String(20))
+    created_by = db.Column(db.JSON)
     updated_at = db.Column(db.DateTime(), default=datetime.utcnow)
-    updated_by = db.Column(db.String(20))
+    updated_by = db.Column(db.JSON)
```

Comment:
Honestly speaking, I don't like the idea of storing it as JSON here, cause of:

1. What id the name of the user changed?
2. Filtering, rql or other filters doesn't support JSON fields filters for now

Let's have it as is for now, but I'll put it as the gap